### PR TITLE
Capstone Project

### PR DESCRIPTION
**Pull request by GutemaG**
### Concepts

- HTML 5 semantic tags
- CSS box model
- Flex box and grid layout
- animation
- Media Queries 👍 
- Web accessibility
- DOM manipulation using JavaScript
- JavaScript events
### Additional
- Fetch API to load local data from JSON file to load dynamically. 💯 

Create two pages about and home.

here is the screenshot 
image
image
image
 
image

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @GutemaG, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Ouch-gifs-38715704-365-199

You have submitted a project not that's not using the updated [readme template](https://github.com/microverseinc/readme-template) for documentation. Kindly utilize the template for a better documented project.  Check the announcement channel on slack for more information about the new template.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -57,174 +36,98 @@ After you're finished please remove all the comments and instructions!
...
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://gutemag.github.io/module-1-capstone/)
 
+## Video Description <a name="video-description"></a>
+- [Live Demo Link](https://gutemag.github.io/module-1-capstone/)
```
Comment (OLIPLICHE) [inline README.md]: - Please record a video presenting the features of the project. The maximum length of the video is 5 minutes, The video must contain a highlight about a piece of the code you find interesting or is proud of making. **All these are the projection requirements section of the exercise.**

File: css/about/footer.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: var(--primary-color);
+}
+
+footer img {
+  width: 200px;
+  background-color: #ccc;
+  border-radius: 5px;
+}
+
+footer p {
```
Comment (OLIPLICHE) [inline css/about/footer.css]: - **About: Mobile size**:
- It has a white background.

Capture1
- See Your Design:

Capture1

File: about.html [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          natus.
+        </p>
+        <ul class="past-summits">
+          <li class="past-summit">
+            <img src="assets/topics/AI-Agri.png" alt="AI in Agriculture" >
+            <div class="overlay"></div>
+            <div class="past-summit-content">
+              <h3>2022</h3>
+              <p>AI in Agriculture</p>
+            </div>
```
Comment (OLIPLICHE) [inline about.html:85]: - **About: Mobile size**: 
The pages should look almost identical to the original design. Here you left behind the partner section
- Original design:

- Your Design:

Capture1
NB: Open this Link to see the 👉  [Original Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)

File: css/header.css [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+
+.nav .nav-items {
+  display: none;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background-color: var(--secondary-color);
+  flex-direction: column;
```
Comment (OLIPLICHE) [inline css/header.css]: - About and Home: Mobile size: 
- The pages should look almost identical to the original design: colors, text, backgrounds, etc ...
- 
Capture
- **Yours, you need to change the top color. much the color to the main top**

File: index.html [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+            <p class="program-description">
+              Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempora,
+              accusantium.
+            </p>
+          </li>
+        </ul>
+        <button class="join-btn btn btn-secondary">Join PanAfrica 2022</button>
+        <a href="#" class="see-whole-program-btn">See The Whole Program</a>
+      </section>
+      <section class="feature-speaker">
```
Comment (OLIPLICHE) [inline index.html:116]: - **Desktop: Home**: It contains a bottom section with a column that includes **six individual articles, each with an image,**
- Kindly remove some and make it six (6):

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @GutemaG, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- GitHub flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️
- Here is the [Link to the Project Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GutemaG** [comment]
hi @OLIPLICHE, thank you for brief comments on my code! Now I hope I make it right can you check it for me please!

[try live](https://gutemag.github.io/module-1-capstone/)

- Video 
image
- About mobile view 
image
- Nav color for mobile view 
image
- speaker part 
image

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @GutemaG 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the changes provided by the previous reviewer ✅
- Your project is professional ✅
- Good video presentation ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [ OPTOINAL ] I noticed that the MIT link in your README file is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

> Also, I suggest removing this `NOTE ...` recommendation from your README file 👍

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**